docs/Installation/CLI.md

```diff
@@ -3,9 +3,9 @@
 A command line interface to perform functions not found in the administrative GUI is available. It can also be used as part of a scripting process, e.g. bulk export/upload etc. This utility can be installed on Linux via pip (package management system to install Python software packages).
 
 ### Requirements:
-It is not necessary for this program to be installed on the Papertrail server that you intend on connecting to. However, the following packages need to be installed on the Linux host where you intend on using the `pt` CLI command:
-* Python
-* Pip
+It is not necessary for this program to be installed on the Papertrail server that you intend on connecting to. However, the following packages need to be installed on the Linux host where you intend on using the `pt` CLI command:<br>
+* Python<br>
```

Comment:
The best practise for md is a double space at the end of the line for a newline - but <br> works as well